_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -177,7 +177,7 @@ This `input` element does not have an [explicit role][] of `textbox`, but the `a
 
 #### Failed Example 1
 
-The `aria-sort` [property][] may not be used on a [semantic][semantic role]]  `button`.
+The `aria-sort` [property][] may not be used on a [semantic][semantic role]] `button`.
```

Comment:
```suggestion
The `aria-sort` [property][] may not be used on a [semantic][semantic role] `button`.
```
(I'm also not sure these typo are related with what the PR says it does and whether it should be part of another PR instead)